sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -55,6 +57,12 @@ def __init__(self, topic, subscription=None, id_label=None):
         case, deduplication of the stream will be strictly best effort.
     """
     super(ReadStringsFromPubSub, self).__init__()
+    if topic and subscription:
```

Comment:
Unfortunately this'll break the direct runner PubSub, but I'll get a fix out for that.